bin/cyclone/BUCK

```diff
@@ -1,4 +1,30 @@
-load("@prelude-si//:macros.bzl", "rust_binary")
+load(
+    "@prelude-si//:macros.bzl",
+    "docker_image",
+    "export_file",
+    "rust_binary",
+    "shellcheck",
+    "shfmt_check",
+    "test_suite",
+)
+
+test_suite(
+    name = "check-lint",
+    tests = [
+        ":check-lint-rust-bin",
+        ":check-lint-rust-unit",
+        ":check-lint-shell",
+    ],
+)
+
+test_suite(
+    name = "check-format",
+    tests = [
+        ":check-format-rust",
+        ":check-format-shell",
+    ],
+)
```

Comment:
I'm surprised these doesn't come along with our `rust_binary` wrapper in `prelude-si`. I'm AFK at the moment, but will look later.